### PR TITLE
feat: remove functions, interfaces, tests

### PR DESCRIPTION
**Pull request by dbouwman**
1. Description:

Remove old `_search*` functions, backing helpers, interfaces and tests

1. Instructions for testing:

run tests

1. [x] used semantic commit messages
  
1. [x] PR title follows semantic commit format (**CRITICAL** if the title is not in a semantic format, the release automation will not run!)

### CODE REVIEW

File: packages/common/src/search/_internal/hubSearchItems.ts [hunk 1]
```diff
@@ -175,17 +75,17 @@ export function hubContentToSearchResult(
...
+//   hasNext: boolean;
+//   next: number;
+//   aggregations: Array<{
+//     field: string;
+//     values: Array<{
+//       value: any;
+//       count: number;
+//     }>;
+//   }>;
+// }
```
Comment (tomwayson) [inline packages/common/src/search/_internal/hubSearchItems.ts:91]: did you mean to delete this?

File: packages/common/src/search/types/types.ts [hunk 2]
```diff
@@ -1,205 +1,5 @@
...
-}
-
-export interface IEventFilterDefinition {
-  created?: IDateRange<number> | IRelativeDate;
-  modified?: IDateRange<number> | IRelativeDate;
-  orgid?: string | string[] | IMatchOptions;
-  title?: string | string[] | IMatchOptions;
-}
+// import { UserSession } from "@esri/arcgis-rest-auth";
+// import { IHubContent, AccessLevel } from "../..";
```
Comment (tomwayson) [inline packages/common/src/search/types/types.ts:2]: more commented code that could be deleted

**Review by tomwayson — APPROVED** [review]
nice work!

just a couple of sets of commented code that look they could be deleted

File: packages/common/src/search/types/types.ts (hunk 2 above)
Comment (dbouwman) [reply packages/common/src/search/types/types.ts]: removed

**Pull request merged**